WeakAuras/WeakAuras.lua

```diff
@@ -2538,11 +2538,22 @@ function Private.AddMany(tbl, takeSnapshots)
   coroutine.yield()
   local groups = {}
   for _, data in ipairs(order) do
-    WeakAuras.Add(data, takeSnapshots);
-    coroutine.yield()
+    WeakAuras.PreAdd(data)
     if data.regionType == "dynamicgroup" or data.regionType == "group" then
       groups[data] = true
     end
+    coroutine.yield()
+  end
+
+  for data in pairs(groups) do
+    WeakAuras.PreAdd(data)
+    coroutine.yield()
+  end
```

Comment:
Is there a need to call PreAdd twice on groups?

Reply:
No that's not necessary.